internal/pkg/defaults/defaults.go

```diff
@@ -9,36 +9,42 @@ import (
 
 	"github.com/ghodss/yaml"
 	"github.com/gobuffalo/packr"
+	"github.com/kiegroup/kie-cloud-operator/internal/pkg/shared"
 	"github.com/kiegroup/kie-cloud-operator/pkg/apis/kiegroup/v1"
 	opv1 "github.com/kiegroup/kie-cloud-operator/pkg/apis/kiegroup/v1"
+
 	"github.com/sirupsen/logrus"
 )
 
-func GetEnvironment(cr *opv1.App) (v1.Environment, error) {
-	env := v1.Environment{}
+func GetEnvironment(cr *opv1.App) (v1.Environment, []byte, error) {
+	var env v1.Environment
 	// default to '1' Kie Server
 	if cr.Spec.NumKieServers == 0 {
 		cr.Spec.NumKieServers = 1
 	}
 
+	// pw := []byte("mykeystorepass")
+	pw := shared.GeneratePassword(8)
```

Comment:
Same here: Please use "password", or if you can't, passwd or similar that is more clear and less ambiguous.